src/robotlibcore.py

```diff
@@ -22,17 +22,7 @@
 import inspect
 import sys
 
-try:
-    from robot.api.deco import keyword
-except ImportError:  # Support RF < 2.9
-    def keyword(name=None, tags=()):
-        if callable(name):
-            return keyword()(name)
-        def decorator(func):
-            func.robot_name = name
-            func.robot_tags = tags
-            return func
-        return decorator
+from robot.api.deco import keyword
```

Comment:
My initial reaction was that this isn't a good idea, but after thinking this a bit more I guess this is fine.

1. Not supporting RF < 2.9 ought to be fine.

2. If we expect `keyword` to set `robot_types`, then the code will only work with RF 3.1+. That would be a problem.

3. The code seems to use `hasattr` so `robot_types` being missing ought to be fine.

4. Only RF 3.1+ is interested in types and calls `get_keyword_types`, so `robot_types` being missing earlier versions shouldn't have any effect.

Reply:
I am confused, should I do something actually? 

But should we say somewhere, in the README example, that starting from next release RF 2.9 is suported? And should we support only RF 3.0 and 3.1+ ?